### PR TITLE
Add PDS Interop projects

### PR DESCRIPTION
**Pull request by Potherca**
Adds mention of the following PDS Interop projects:

- Solid test-suites
- Solid Nextcloud
- PHP Solid Server (or should that be "Solid PHP Server"?)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -78,6 +82,12 @@ be any production-ready Pod providers. Only the one prototypes in multiple place
 - [solid.community](https://solid.community/) - Hosted by (?)) on (?)).
 - [Solid Web](https://solidweb.org/) - Hosted by Matthias Evering on (?)
 
+## Servers
+
+- [PHP Solid Server](https://github.com/pdsinterop/solid-php-server)
```
Comment (michielbdejong) [inline README.md]: If it's "PHP Solid Server" then why not call the repo "php-solid-server" instead of "solid-php-server"?

File: README.md [hunk 2]
```diff
@@ -63,6 +65,8 @@ Things that explain how Solid works (or how to work with Solid).
 
 Things built on top of solid or that plug Solid into other things.
 
+- [Solid Nextcloud](https://pdsinterop.org/solid-nextcloud) - Turns a Nextcloud Server into a Solid Pod
+  ([repo](https://github.com/pdsinterop/solid-nextcloud)).
```
Comment (michielbdejong) [inline README.md:69]: Second link to the repo is maybe superfluous?

File: README.md [hunk 3]
```diff
@@ -78,6 +82,12 @@ be any production-ready Pod providers. Only the one prototypes in multiple place
 - [solid.community](https://solid.community/) - Hosted by (?)) on (?)).
 - [Solid Web](https://solidweb.org/) - Hosted by Matthias Evering on (?)
 
+## Servers
+
+- [PHP Solid Server](https://github.com/pdsinterop/solid-php-server)
+  \- Solid Server written in PHP
+  ([repo](https://github.com/pdsinterop/solid-php-server))
```
Comment (michielbdejong) [inline README.md]: Repo link is maybe superfluous if it's the same as the main project link?

File: README.md (hunk 1 above)
Comment (Potherca) [reply README.md]: Good point! I've renamed the repo. Will change URL here.

File: README.md (hunk 2 above)
Comment (Potherca) [reply README.md:69]: Not sure that I agree. The main link https://pdsinterop.org/solid-nextcloud is more project-related / consumer-facing. The repo is added as a bonus. 

The setup seems to work well with the current content (see https://pdsinterop.org/awesome-solid/).

File: README.md (hunk 3 above)
Comment (Potherca) [reply README.md]: Good catch! My bad :disappointed_relieved: , should be https://pdsinterop.org/php-solid-server/. Will fix.

**Pull request merged**